plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -246,18 +246,39 @@ export class ElasticSearchSearchEngine implements SearchEngine {
           'Failed to add filters to query. Unrecognized filter type',
         );
       });
-    const esbQuery = isBlank(term)
-      ? esb.matchAllQuery()
-      : esb
-          .multiMatchQuery(['*'], term)
-          .fuzziness('auto')
-          .minimumShouldMatch(1);
+
+    const esbQueries = [];
+    const phraseTerms = term.match(/"[^"]*"/g);
+
+    if (isBlank(term)) {
+      const esbQuery = esb.matchAllQuery();
+      esbQueries.push(esbQuery);
+    } else if (phraseTerms && phraseTerms.length > 0) {
+      let restTerm = term;
+      for (const phraseTerm of phraseTerms) {
+        restTerm = restTerm.replace(phraseTerm, '');
+        const esbPhraseQuery = esb
+          .multiMatchQuery(['*'], phraseTerm.replace(/"/g, ''))
+          .type('phrase');
+        esbQueries.push(esbPhraseQuery);
+      }
+      if (restTerm?.length > 0) {
+        const esbRestQuery = esb
+          .multiMatchQuery(['*'], restTerm.trim())
+          .fuzziness('auto');
+        esbQueries.push(esbRestQuery);
+      }
+    } else {
+      const esbQuery = esb.multiMatchQuery(['*'], term).fuzziness('auto');
+      esbQueries.push(esbQuery);
+    }
+
     const pageSize = query.pageLimit || 25;
     const { page } = decodePageCursor(pageCursor);
 
     let esbRequestBodySearch = esb
       .requestBodySearch()
-      .query(esb.boolQuery().filter(filter).must([esbQuery]))
+      .query(esb.boolQuery().filter(filter).should(esbQueries))
```

Comment:
From [Lucene Boolean Clause docs](https://lucene.apache.org/core/5_0_0/core/org/apache/lucene/search/BooleanClause.Occur.html)
> public static final [BooleanClause.Occur](https://lucene.apache.org/core/5_0_0/core/org/apache/lucene/search/BooleanClause.Occur.html) SHOULD
Use this operator for clauses that should appear in the matching documents. For a BooleanQuery with no MUST clauses one or more SHOULD clauses must match a document for the BooleanQuery to match.

As long as one of the queries in the `esbQueries` match for a document, it will be included. I think this behavior is sensible. For example, if the search term is: `"foo bar" test` , then we would want the search result to include documents that match on either the phrase query or the regular term query or both.